app/components/IndicatorChart/styles.css

```diff
@@ -1,8 +1,7 @@
 .indicator-chart-card {
     display: flex;
     flex-shrink: 0;
-    border: var(--width-separator-thin) solid var(--color-separator-light);
-    border-radius: var(--dui-border-radius-card);
+    border-radius: 0;
```

Comment:
Please add a note here why you had to override the card style.

Reply:
Done